libs/heattracesidesheet/src/lib/ui-sidesheet/HeatedTagsTab.tsx

```diff
@@ -0,0 +1,22 @@
+import { TabTable } from '@cc-components/shared';
+import { heatedTagsColumns } from './heatedTagsColumns';
+import { HeatTraceHeatedTag } from 'libs/heattraceshared/dist/src';
```

Comment:
This import is not valid but will work because its a type, but importing any js would cause an error. Please update import to @cc-components/heattrace-shared

Reply:
Thought i had fixed this. Fixed now